### DISCUSSION TITLE
Stuck at Connecting to radio://0/80/2M in Crazy Radio 2.0 with CrazyFlie 2.1 with AI deck

### DISCUSSION THREAD

**Discussion by VenkatSBitra** [bitcraze/discussions#1582] [Q&A]
I am using a Windows 11 machine. I have only USB 3.0 and 3.1 ports. I get stuck at "Connecting to radio://0/80/2M". I used the latest CF2 firmware in the GitHub repository. 

I get the following in the terminal when I start connecting:

INFO:cflib.crazyflie:Callback->Connection initialized[radio://0/80/2M]
INFO:cflib.crazyflie:We are connected[radio://0/80/2M], request connection setup
INFO:cflib.crazyflie.platformservice:Request _request_protocol_version()
INFO:cflib.crazyflie:Callback->Connected to [radio://0/80/2M]
INFO:cflib.crazyflie.platformservice:_crt_service_callback
INFO:cflib.crazyflie.platformservice:Request protocol version
INFO:cflib.crazyflie.platformservice:_platform_callback
INFO:cflib.crazyflie.platformservice:Protocol version (platform): 7
INFO:cflib.crazyflie.toc:TOC for port [5] found in cache
INFO:cflib.crazyflie:Log TOC finished updating

Can anyone provide me possible causes for the same. The LED of the CrazyRadio stays green when in connecting mode. 

![image](https://github.com/user-attachments/assets/98b38ee4-8510-4bbd-9fa0-968838b063c7)
![image](https://github.com/user-attachments/assets/219ae611-2abb-40c7-b346-b1a03e6695f1)

**Comment by knmcguire** [comment]
We need some more information of your situation:

- Are you able to connect without the aideck?
- Do you get any readout on the console tab?

**Reply by VenkatSBitra** [reply]
I have uploaded my console logs as shown.  Also, I updated the bootloader as mentioned in the guide and my CrazyFlie no longer connects to the mobile device via BLE. I want to know if there is a docker container on Windows so that it makes it easy to prepare the setup.

**Reply by knmcguire** [reply]
Hi!

That is not the console tab of the cfclient, but the terminal output of where the cfclient is running from. I'm refering to this one: https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/console_tab/

Also could you clarify what you mean with updating the bootloader? Because that is not what we recommend as of standard. Could you share the section of the guide you were following?

**Reply by gemenerik** [reply]
I think they meant updating the AI-deck bootloader
